compress/jbig2_compress_image.go

```diff
@@ -0,0 +1,88 @@
+package main
```

Comment:
Add a comment at top similar to other packages with an explanation what this example showcases.
Maybe this should be in a subfolder?
Either compress/jbig2
or image/jbig2 ?

Reply:
Already done for all files.